packages/botonic-dx-bundler-rspack/baseline/rspack.config.ts

```diff
@@ -0,0 +1,380 @@
+import { Configuration } from '@rspack/cli'
+import * as rspack from '@rspack/core'
+import ReactRefreshPlugin from '@rspack/plugin-react-refresh'
+import HtmlWebpackPlugin from 'html-webpack-plugin'
+import NodePolyfillPlugin from 'node-polyfill-webpack-plugin'
+import path from 'path'
+
+const ROOT_PATH = path.resolve(__dirname, 'src')
+const OUTPUT_PATH = path.resolve(__dirname, 'dist')
+const NODE_MODULES_PATH = path.resolve(__dirname, 'node_modules')
+const WEBVIEWS_PATH = path.resolve(OUTPUT_PATH, 'webviews')
+const BOTONIC_PATH = path.resolve(NODE_MODULES_PATH, '@botonic', 'react')
+
+enum BotonicTarget {
+  ALL = 'all',
+  DEV = 'dev',
+  NODE = 'node',
+  WEBVIEWS = 'webviews',
+  WEBCHAT = 'webchat',
+}
+
+const WEBPACK_ENTRIES_DIRNAME = 'webpack-entries'
+const WEBPACK_ENTRIES = {
+  DEV: 'dev-entry.ts',
+  NODE: 'node-entry.ts',
+  WEBCHAT: 'webchat-entry.ts',
+  WEBVIEWS: 'webviews-entry.ts',
+}
+
+const TEMPLATES = {
+  WEBCHAT: 'webchat.template.html',
+  WEBVIEWS: 'webview.template.html',
+}
+
+const UMD_LIBRARY_TARGET = 'umd'
+
+const LIBRARY_NAME = {
+  BOTONIC: 'Botonic',
+  WEBVIEW: 'BotonicWebview',
+  BOT: 'bot',
+}
+
+const FILENAME = {
+  WEBCHAT: 'webchat.botonic.js',
+  WEBVIEWS: 'webviews.js',
+  BOT: 'bot.js',
+}
+
+enum Mode {
+  development = 'development',
+  production = 'production',
+}
+
+const HUBTYPE_DEFAULTS = {
+  API_URL: 'https://api.hubtype.com',
+  WEBCHAT_PUSHER_KEY: '434ca667c8e6cb3f641c', // pragma: allowlist secret
+}
+
+const CONFIG_ENVIRONMENTS = ['production', 'staging', 'local']
+
+const resolveConfig = {
+  extensions: ['.*', '.js', '.jsx', '.ts', '.tsx'],
+  alias: {
+    BotonicProject: ROOT_PATH,
+    react: path.resolve('./node_modules/react'),
+    // 'styled-components': path.resolve('./node_modules/styled-components'),
```

Comment:
remove this line? otherwise, if this is left intentionally add a comment next to specify why